agent/bench-scripts/unittests

```diff
@@ -42,6 +42,10 @@ for script in ${_tdir}/../util-scripts/pbench-* ; do
     cp ${script} ${_testopt}/util-scripts/
     let res=res+${?}
 done
+cp -a $_tdir/../../bin/getconf.py $_testopt/common/bin
+let res=res+${?}
+cp -a $_tdir/../../lib/configtools/__init__.py $_testopt/common/lib/configtools
```

Comment:
Here do  `cp -a $_tdir/../../lib/* $_testopt/common/lib/` - again, just to make sure that when we add additional libraries, we'll copy them all. This also implies that when you do the mkdir on line 27, you only need to create `$_testopt/common/lib` - the rest of the hierarchy will be created by the `cp -a`.